storybook/.storybook/dark.css

```diff
@@ -0,0 +1,100 @@
+@scope (.dark-scheme) {
```

Comment:
need to create an issue to have these files being generated/exported from main package

Reply:
I listed it in the epic https://github.com/containers/podman-desktop/issues/6338, but this is not an issue yet